zone/inventory.cpp

```
@@ -2227,31 +2276,41 @@ bool Client::SwapItem(MoveItem_Struct* move_in) {
 	}
 
 	// Step 7: Save change to the database
+	bool source_saved = false;
 	if (src_slot_id == EQ::invslot::slotCursor) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)